MultiSlider.js

```diff
@@ -131,6 +131,14 @@ export default class MultiSlider extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    if(nextProps.min !== this.props.min && nextProps.max !== this.props.max){
```

Comment:
All of this code is available a few lines down in this already existing method. Whats the reason for introducing another copy paste of it? I mean cWRP already checks for changes in min and max? on Line 145

Reply:
Oh weird, in the version I have in my app that code doesn't exist. I must have an older version? I only got it last week though. I will update and see if the fix is even necessary.

Reply:
Is this new version pushed to npm? It doesn't look like it.

Reply:
Feel free to try it out with inserting ptomasroos/react-native-multi-slider#2f83b6b8b700c0e6ace77c74b8af8a7d2ad83ba0 into your package json instead of a the version number. And see if it resolves your issue